### ISSUE TITLE
Query parameters not taken into account

### ISSUE THREAD

**Issue by toddkazakov** [ctimmerm/axios-mock-adapter#39]
Tracked down the issue to https://github.com/ctimmerm/axios-mock-adapter/pull/32/files#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R32

The problem is that method is all uppercase

**Comment by i-aakopian** [comment]
+1. Also experiencing this issue with a simple example

```javascript
mock.onGet('http://www.mocky.io/v2/58c097970f0000de0f5ee482', { params: { test: 1} }).reply(200, {test: 123});
```

Throws a 404 for me.

**Comment by GodOfGrandeur** [comment]
The problem here is simple... 

The latest release doesn't include the latest changes for params support. A new release and NPM publish is needed. Can this be arranged?

**Comment by ctimmerm** [comment]
Fixed in 6b6fc3d1872ee8ab23eca00bed19f0c5e527e7e0
